couchbase/search/v1/search.proto

```diff
@@ -265,19 +265,36 @@ message SearchQueryResponse {
   }
 
   message Facet {
-    FacetResultType facet_result_type = 1;
     string name = 2;
     string field = 3;
     int64 total = 4;
     int64 missing = 5;
     int64 other = 6;
+    repeated Terms terms = 7;      
+    repeated NumericRange numeric_ranges = 8;
+    repeated DateRange date_ranges = 9;
```

Comment:
Protobuf doesn't support keywords (repeated, optional...) inside oneof, so I went with this way instead.

Reply:
I think it'd look like https://github.com/couchbase/protostellar/pull/31/files but FacetResult etc...?